### PR TITLE
fix(tests): fix test warnings

### PR DESCRIPTION
**Pull request by acateland**
**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)
test had lot of warnings

**What is the new behavior?**
removed them
reformated code according to prettier rules

**Does this PR introduce a breaking change?**

- [ ] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration
path for existing applications: ...

**Other information**:

### CODE REVIEW

File: src/actions/node.actions.test.js [hunk 1]
```diff
@@ -5,130 +5,125 @@ import { Map, OrderedMap } from 'immutable';
...
-describe('Check that node action creators generate proper' +
-	' action objects and perform checking', () => {
-	it('addNode generate action with 0 configuration', () => {
-		const store = mockStore({
-			flowDesigner: {
-				nodes: new Map({}),
-			},
-		});
+describe(
+	'Check that node action creators generate proper' + ' action objects and perform checking',
```
Comment (talend-glorieux) [inline src/actions/node.actions.test.js]: Why the `+`?

File: src/actions/port.actions.test.js [hunk 2]
```diff
@@ -9,21 +8,19 @@ import { PORT_SINK } from '../constants/flowdesigner.constants';
...
-	' action objects and perform checking', () => {
-	it('addPort', () => {
-		const store = mockStore({
-			flowDesigner: {
-				nodes: new Map({ nodeId: { id: 'nodeId', nodeType: 'type' } }),
-				ports: new Map(),
-			},
-		});
+describe(
+	'Check that port action creators generate proper' + ' action objects and perform checking',
```
Comment (talend-glorieux) [inline src/actions/port.actions.test.js]: Why the `+`?

File: src/actions/node.actions.test.js (hunk 1 above)
Comment (acateland) [reply src/actions/node.actions.test.js]: legacy

File: src/actions/port.actions.test.js (hunk 2 above)
Comment (acateland) [reply src/actions/port.actions.test.js]: legacy

**Pull request merged**